architecture/rosa-architecture-models.adoc

```diff
@@ -23,8 +23,14 @@ endif::openshift-rosa-hcp[]
 
 * xref:../rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.adoc#rosa-policy-security-regulation-compliance_rosa-policy-process-security[Security and regulation compliance]
 
+ifdef::openshift-rosa-hcp[]
 include::modules/rosa-hcp-architecture.adoc[leveloffset=+1]
+endif::openshift-rosa-hcp[]
```

Comment:
There are a few "ROSAs" in this module: https://99613--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/rosa_architecture/rosa-architecture-models.html#rosa-hcp-architecture_rosa-architecture-models

<img width="906" height="431" alt="image" src="https://github.com/user-attachments/assets/69ccf049-25aa-44ed-932b-26422fca6794" />